IoTBlockly/ChakraHost/ChakraHostAsync.cs

```diff
@@ -90,109 +96,69 @@ public string dispose()
             return "";
         }
 
-        public void runScript(string script)
+
+        public void ScriptProcessor()
```

Comment:
Ah capitalised following C# method naming conventions, but am easy:)  init() is called inside the try catch block after the waitone.